src/coreclr/nativeaot/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -761,8 +761,124 @@ void Registers_REGDISPLAY::setVectorRegister(int num, libunwind::v128 value)
 
 #endif // TARGET_ARM64
 
+// the cache relies on "endless" 64bit version counter
+#if TARGET_64BIT
+struct ProcInfoCacheEntry
+{
+    PCODE pc;
+    volatile size_t version;
+
+    //    unw_proc_info_t procInfo;
+    uint32_t    start_offset;     /* start address of function relative to pc */
+    uint32_t    end_offset;       /* end address of function relative to pc */
+
+    uint32_t    format;           /* compact unwind encoding, or zero if none */
+    uint32_t    unwind_info_size; /* size of DWARF unwind info, or zero if none */
+    unw_word_t  lsda;             /* address of language specific data area, */
+    unw_word_t  handler;          /* personality routine, or zero if not used */
```

Comment:
I wonder if personality routine is actually used in our scenarios or can be set to 0

Reply:
It looks it is not used.